src/applications/disability-benefits/2346/schemas/2346-schema.json

```diff
@@ -229,68 +250,131 @@
         },
         "country": {
           "type": "string",
-          "enum": ["USA", "CAN", "MEX"]
+          "default": "USA",
+          "enum": [
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/11999/files#r406813416